ledger/src/bank_forks_utils.rs

```diff
@@ -244,8 +244,8 @@ fn bank_forks_from_snapshot(
             .map(SnapshotArchiveInfoGetter::slot)
             .unwrap_or(0),
     );
-    let latest_bank_snapshot =
-        snapshot_utils::get_highest_bank_snapshot_post(&snapshot_config.bank_snapshots_dir);
+
+    let latest_bank_snapshot = snapshot_utils::get_highest_loadable_bank_snapshot(snapshot_config);
```

Comment:
This strikes me as a bit odd as well, though not much different than previously.
We're reading & checking the bank snapshot even if we're `Always` using archives.

Reply:
Ended up making my review comment about this, so can just ignore this individual comment